lib/loaders/loaders_with_authentication/impulse.js

```diff
@@ -30,5 +32,13 @@ export default (accessToken, userID) => {
       },
       { method: "POST" }
     ),
+    markMessageReadLoader: impulseLoader(
+      "events",
+      {
+        event: "open",
+        timestamp: moment().unix(),
```

Comment:
We generally send back unix timestamps (integers) to API's expecting date/time fields.